res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1572,60 +1810,25 @@ PioneerDDJSX.playLed = function(value, group, control) {
     PioneerDDJSX.nonPadLedControl(group, PioneerDDJSX.nonPadLeds.shiftPlay, value);
 };
 
-PioneerDDJSX.playFromCuePoint = false;
-
-PioneerDDJSX.playIllumination = function(value, group, control) {
+PioneerDDJSX.wheelLeds = function(value, group, control) {
     var deck = PioneerDDJSX.channelGroups[group],
-        cue = PioneerDDJSX.cuePressed[deck],
-        trackLoaded = 0;
+        duration = engine.getValue(group, "duration"),
+        remainingTime = duration - (value * duration),
+        speed = 0x28,
+        wheelPos = 0x01;
 
-    trackLoaded = engine.getValue(group, "track_loaded");
-    if (!trackLoaded) {
-        return;
+    if (value >= 0) {
```

Comment:
I read that but it doesn't make any sense to me to let the wheel LEDs rotate when seeking before the beginning of the track. Not good?